extensions/pom.xml

```diff
@@ -37,5 +37,6 @@
         <module>jackson</module>
         <module>orgjson</module>
         <module>gson</module>
+        <module>jsonb</module>
```

Comment:
This is going to fail on the Java 7 build.
We are currently compiling for 7 for legacy (Android) support.

This can probably be pushed into a profile for java 1.8+ though